src/Subscriptions/Events/EchoSubscriptionEvent.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Nuwave\Lighthouse\Subscriptions\Events;
+
+use Illuminate\Broadcasting\PresenceChannel;
+use Illuminate\Contracts\Broadcasting\ShouldBroadcast;
+
+class EchoSubscriptionEvent implements ShouldBroadcast
+{
+    /**
+     * @var string
+     */
+    public $channel;
+
+    /**
+     * @var mixed
```

Comment:
StyleCI removes those if left plain. We solve this by adding a quick explanation to the type:

```suggestion
     * @var mixed The data to broadcast.
```

Reply:
Ah ok I was wondering... StyleCI wants them removed and phpstan wants them there.

Reply:
I was annoyed with that at first, but since then have come to peace with it. It is actually reasonable to document those untyped parts a bit, at least explaining why we can not have a type for it.